Day2/Exercise2/Solution.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+func expectedRating(val float64, wg *sync.WaitGroup, sum *float64){
+	defer wg.Done()
+	time.Sleep(time.Second*time.Duration(rand.Intn(2)))
+	*sum=*sum+val
+}
+func main () {
+
+	var wg sync.WaitGroup
+	var students[200] float64
```

Comment:
Instead of using an array to allocate ratings , do it directly . Yes , this would've been the ideal case but for this particular question we don't require allocation , so can be done directly